### ISSUE TITLE
'amass -d doctorlan.com' fails to find subdomains

### ISSUE THREAD

**Issue by yurivict** [OWASP/Amass#32]
2.6.x found them, but 2.7.0 returns nothing.

**Comment by caffix** [comment]
Important fixes were made over the weekend. Please try again with version 2.7.3.

**Comment by yurivict** [comment]
Ok, once 2.7.3 is available.

**Comment by caffix** [comment]
Version 2.7.4 is now available.

**Comment by yurivict** [comment]
There's no such tag: https://github.com/OWASP/Amass/tags

**Comment by caffix** [comment]
Have the fixes made to the project taken care of the issues you were experiencing? Thank you in advance for your feedback.

**Comment by kozooh** [comment]
Hi, just checked on Kali Linux and got empty results:
```
root@kali:~# amass -d doctorlan.com
root@kali:~# amass -version
version 2.7.4
root@kali:~# uname -a
Linux kali 4.17.0-kali1-amd64 #1 SMP Debian 4.17.8-1kali1 (2018-07-24) x86_64 GNU/Linux
```

**Comment by caffix** [comment]
@kozooh  What method of installation are you using?

**Comment by kozooh** [comment]
@caffix I've used Snap.

**Comment by yurivict** [comment]
The latest master fixed the problem for me.

**Comment by caffix** [comment]
@yurivict Great to hear that you're no longer experiencing any problems.

@kozooh I'll be pushing out the version 2.7.5 snap package tonight that is likely to fix the problem you're having right now.

Thank you both for reaching out!

**Comment by caffix** [comment]
@kozooh Please try again with version 2.7.5 and let me know if it fixed the problem you were having. Thanks!

**Comment by kozooh** [comment]
@caffix I updated the package and it still doesn't work but... I think the cause may be that I'm using Kali Linux in VMware Workstation since I did some checking on Kali running in AWS environment and Amass works perfectly there. 

Did you, by any chance, tested Amass in a virtual environment? For now, I noticed that when running `tcpdump`, simply all of the packets that Amass generated are "dropped by kernel". Sadly I don't have any experience with Go and I don't even know how could I debug the issue.

This is awesome tool, btw :)
